app/assets/stylesheets/mo/_status_light.scss

```diff
@@ -0,0 +1,85 @@
+// ./app/assets/stylesheets/mo/_status_light.scss
+
+.status-light-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+  padding: 20px;
+}
+
+.status-indicator {
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+  border: 2px solid #ccc;
+  background: #f8f9fa;
+  transition: all 0.3s ease;
+  flex-shrink: 0;
+
+  &.red {
+    background: #dc3545;
+    border-color: #dc3545;
```

Comment:
Instead of hardcoding all these colors, they should maybe be selected from the theme variables palette.

Especially the text colors like `status-text` should use a background and text color combination defined on the theme to be legible, or else this will probably look weird in other themes.